source/versions/filenames.rst

```diff
@@ -0,0 +1,117 @@
+| ``LatestAPI``        | The minor version of SpongeAPI implemented by this file (the ``Y`` in                         |
+| ``MinorRelease``     | :ref:`semantic versioning<sem-ver>`).                                                         |
++----------------------+-----------------------------------------------------------------------------------------------+
+| ``Recommended``      | The released version of the implementation when **not** followed by ``-RC<BuildNumber>.``     |
+| ``Version``          |                                                                                               |
++----------------------+-----------------------------------------------------------------------------------------------+
+| ``-RC``              | The build number in development for the next recommended release. When a build number is      |
+| ``<BuildNumber>``    | present, the ``RecommendedVersion`` has **not** been released yet.                            |
++----------------------+-----------------------------------------------------------------------------------------------+
+
+Examples
+~~~~~~~~
+
+SpongeForge Jar files will always follow this naming scheme to allow you to easily identify compatibility.
+
+For example, the file name ``spongeforge-1.12.2-2705-7.1.4.jar`` is compatible with Minecraft version ``1.12.2``, was 
+built with Forge ``14.23.4.2705`` (Build ``2705``), guarantees compatibility with SpongeAPI ``7.1.0``, is a recommended 
+version, and is the ``4`` th release of SpongeForge with this API.
+
+Another example is the file name ``spongeforge-1.12.2-2705-7.1.5-RC3449.jar``. This file is compatible with Minecraft 
+version ``1.12.2``, was built with Forge ``14.23.4.2705`` (Build ``2705``), guarantees compatibility with SpongeAPI 
+``7.1.0``, is not a recommended version, and is build number ``3449`` in development, which will be the ``5`` th 
+release of SpongeForge once this version is released. 
+
+.. note::
+
+    Normal Forge mods can usually run on any build of Forge for a given Minecraft version (e.g. 1.8.0) without any
+    problem. However, SpongeForge needs to access, among other things, internal parts of Forge, which most mods
+    shouldn’t be touching, let alone modifying as Sponge does. Since Forge is free to change their internal code
+    whenever they want to, its normal guarantee of backwards-compatibility doesn’t apply to SpongeForge. Feel free to
+    use more recent versions of Forge, than the one used for SpongeForge, but we can't always guarantee compatibility.
+
+.. warning::
+
+    When investigating crash issues, you can freely try newer versions of Forge than listed on the SpongeForge Jar.
+    However, it is recommended to also check with the matching version, to make sure your issue is not related to a
+    version mismatch.
+    Even though there will be no guarantee of compatibility, please report any breakage to the issue tracker, so that
```

Comment:
See above.

This is misleading, if you're not using the recommended forge version then there's nothing to fix as we won't break compatibility with the recommended build.

Reply:
I agree (with my caveats, above), [but I just want to point out this was a copy/paste](https://github.com/SpongePowered/SpongeDocs/pull/785/files/3925d9afa55f368d4d6300331e443d9da05e4f19#diff-1b21b02d7e6287095c0a769eab09ebc1L64). This is a repeat of what's already been said anyway, so this entire warning could probably be removed.

Reply:
Warning removed; note moved to the top and combined with the introduction.